charts/tfy-karpenter-config/values.yaml

```diff
@@ -18,13 +18,16 @@ karpenter:
       - key: kubernetes.io/arch
         operator: In
         values: ["amd64"]
+      - key: karpenter.k8s.aws/instance-family
```

Comment:
By default instance-family is '`c', 'm', 'r'.  [Default Instance type](https://karpenter.sh/v0.26.1/concepts/provisioners/#instance-types)

Reply:
removed for now